application/helpers/admin/export/JsonWriter.php

```diff
@@ -34,9 +34,22 @@ public function init(SurveyObj $survey, $sLanguageCode, FormattingOptions $oOpti
         }
     }
 
-    protected function outputRecord($headers, $values, FormattingOptions $oOptions)
+    protected function outputRecord($headers, $values, FormattingOptions $oOptions, $fieldNames = [])
     {
-        $aJson = array_combine($headers, $values);
+        $uniqueHeaders = [];
+        // Major Problem: We can't just combine headers and values using
+        // array_combine, because same header can be there multiple times.
+        // If a header is not unique, we add the fieldname at the end of the
+        // duplicated header.
+        foreach ($headers as $key => $header) {
+            if (!in_array($header, $uniqueHeaders)) {
+                $uniqueHeaders[$key] = $header;
+            }else{
+                $fieldName = array_key_exists($key, $fieldNames) ? $fieldNames[$key] : '*';
```

Comment:
fallback is a *, not sure if thats a good idea, maybe add a timestamp. Also this should never happen.